charts/ztka/values.yaml

```diff
@@ -296,6 +296,17 @@ kratos:
               oidc:
                 hooks:
                   - hook: session
+          login:
+            ui_url: http://dashboard:3000/login
+            lifespan: 10m
```

Comment:
Just curious, do you have any reason for setting lifespan to 10m?

Reply:
No, this was the default. So did not update. If you have any other value in mind. Do let me know.

Reply:
updated to default